DayCare/admin.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+
+
+# Register your models here.
+from .models import DayCare
+
+
+class DayCareAmid(admin.ModelAdmin):
```

Comment:
I guess its type and it should have been `DayCareAdmid`

Reply:
I'm not sure about it, but it was generated by Django automatically
Maybe we need to ךeave it as it is nowץ